sdk/core/Azure.Core/tests/DisposableHttpPipelineTests.cs

```diff
@@ -44,7 +43,7 @@ private class MockDisposableHttpPipelineTransport : HttpPipelineTransport, IDisp
             public override Request CreateRequest() { throw new NotImplementedException(); }
 
             public bool DisposeCalled;
-            public void Dispose() => DisposeCalled = true;
+            public override void Dispose() => DisposeCalled = true;
```

Comment:
Flagging this as a possible breaking change for Azure.Core consumers, and will investigate and make sure it is addressed prior to merging this PR.